### PR TITLE
Puneet/update to use android x and u7

### PR DESCRIPTION
**Pull request by puneet-pdx**

### CODE REVIEW

**Comment by puneet-pdx** [comment]
@gunt0001 can you please review these changes. The changes are for the app to use runtime Update 7 and Androidx libraries.

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+
+android.enableJetifier=true
+android.useAndroidX=true
```
Comment (changanxian) [inline gradle.properties]: Is this file necessarily kept after you finished the migration?

Comment (puneet-pdx) [reply gradle.properties]: yes, these configuration are required for migrating to AndroidX

**Comment by puneet-pdx** [comment]
@eri9600 can you review.

**Comment by puneet-pdx** [comment]
@esreli do you wanna look at the new release I added to RELEASE.md file before I go ahead and merge this ?

**Comment by esreli** [comment]
@puneet-pdx sure. LGTM.

**Pull request merged**